src/libcollections/range.rs

```diff
@@ -23,13 +23,45 @@ pub trait RangeArgument<T> {
     /// Start index (inclusive)
     ///
     /// Return start value if present, else `None`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(collections)]
+    /// #![feature(collections_range)]
+    ///
+    /// extern crate collections;
+    ///
+    /// # fn main() {
```

Comment:
It shouldn't but it works and it is not (the generation is "clever" *normally*).